tomni/cytosmart_data_format/annotations/polygon/test_polygon_comparing.py

```diff
@@ -0,0 +1,92 @@
+from unittest import TestCase
+
+from tomni.cytosmart_data_format import Point, Polygon
+
+
+class TestPolygonCompare(TestCase):
+    def test_positive_squares(self):
```

Comment:
can you add a complex shape to test the simplification

Reply:
i did not see the star test. but i mean more like a polygon that shows when a point is not removed by simplification